src/components/sidebar/routePathView/RoutePathView.tsx

```diff
@@ -62,143 +45,43 @@ class RoutePathView extends React.Component<IRoutePathViewProps, IRoutePathViewS
-        if (!routePath) return;
-
-        const newRoutePathLink = routeBuilder
-        .to(subSites.newRoutePath, { routeId: routePath.routeId, lineId: routePath.lineId })
-        .toLink();
-
-        this.props.routePathStore!.setRoutePath(this.state.routePath);
-        navigator.goTo(newRoutePathLink);
-
     }
 
     public render(): any {
-        // tslint:disable-next-line:max-line-length
-        const message = 'Reitin suunnalla tallentamattomia muutoksia. Oletko varma, että poistua näkymästä? Tallentamattomat muutokset kumotaan.';
-
         if (this.state.isLoading) {
             return (
                 <div className={s.routePathView}>
                     <Loader size={Loader.MEDIUM}/>
                 </div>
             );
         }
-        if (!this.state.routePath) return 'Error';
+        if (!this.state.routePath) return;
         return (
-        <div className={classnames(s.routePathView, s.form)}>
-            <div className={s.formSection}>
-                <ViewHeader
-                    header={`Reitin suunta`}
-                    closePromptMessage={this.state.hasModifications ? message : undefined}
-                >
-                    <Button
-                        onClick={this.toggleEditing}
-                        type={ButtonType.SQUARE}
-                        text={this.state.isEditingDisabled ? 'Muokkaa' : 'Peruuta'}
+            <div className={s.routePathView}>
+                <div className={s.flexInnerRow}>
+                    <RoutePathViewTabButtons
+                        tabs={['Reitinsuunta', 'Solmut ja linjat']}
+                        selectedTab={this.state.selectedTab}
+                        onClick={this.selectTab}
                     />
-                </ViewHeader>
-            </div>
-            <div className={s.formSection}>
-                <div className={s.topic}>
-                    REITIN OTSIKKOTIEDOT
                 </div>
-                <div className={s.routeInformationContainer}>
-                    <div className={s.flexInnerColumn}>
-                        <div>Reittitunnus</div>
-                        <div>{this.state.routePath.routeId}</div>
-                    </div>
-                    <div className={s.flexInnerColumn}>
-                        <div>Linja</div>
-                        <div>{this.state.routePath.lineId}</div>
-                    </div>
-                    <div className={s.flexInnerColumn}>
-                        <div>Päivityspvm</div>
-                        <Moment
-                            date={this.state.routePath.lastModified}
-                            format='DD.MM.YYYY HH:mm'
-                        />
-                    </div>
-                    <div className={s.flexInnerColumn}>
-                        <div>Päivittäjä</div>
-                        <div>{this.state.routePath.modifiedBy}</div>
+
+                {(this.state.selectedTab === 'Reitinsuunta') ?
```

Comment:
This feels sort of magic numberish. Should it rather be a component specified by key?

Reply:
I don't know if there's a better solution for this. With enums it's maybe a bit less magic numberish now